oci_spec/Cargo.toml

```diff
@@ -12,5 +12,5 @@ serde = { version = "1.0", features = ["derive"] }
 nix = "0.19.1"
 anyhow = "1.0"
 serde_json = "1.0"
-caps = "0.5.1"
+caps = { git = "https://github.com/lucab/caps-rs", features = ["serde_support"] }
```

Comment:
Can you add a rev or tag and comment that you are waiting for caps to be released?